pkg/materialize/utils.go

```diff
@@ -26,7 +26,12 @@ func QualifiedName(fields ...string) string {
 func GetSliceValueString(v []interface{}) []string {
 	var o []string
 	for _, i := range v {
-		o = append(o, i.(string))
+		if i != nil {
+			str, ok := i.(string)
+			if ok {
+				o = append(o, str)
+			}
+		}
```

Comment:
I think we should return an error here rather than silently excluding nil! E.g., IMO it is just a straight error to specify `text_columns = [nil]`, and all we need to do is return a helpful error message rather than panicking. Does that check out for you?

Reply:
Good idea! Just refactored the PR to return an error instead.